src/controllers/index.js

```diff
@@ -174,7 +174,7 @@ const getControlNumber = function(req, res, reqData){
 	if (basicData){
 		db.getApplication(controlNumber, function(err, appl, fileData){
 			if (err){
-				return error.sendError(req, res, 400, 'error getting application from database');
+				return error.sendError(req, res, 404, 'file not found');
```

Comment:
@cameronwolf I had these response code updates in another PR, didn't realize these were merged. Will revisit.